sematic/tests/integration/test_pip_install.sh

```diff
@@ -2,14 +2,27 @@
 
 pip install virtualenv
 
-VENV_NAME=$RANDOM
+if [ -z "$CI_VENV_NAME" ]
+then
+    VENV_NAME=$RANDOM
+else
+    VENV_NAME=$CI_VENV_NAME
```

Comment:
If this file is not supposed to run except in the CI environment, then we can simplify this.